sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -232,6 +232,22 @@ class WholeStageCodegenSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  ignore("SPARK-24901 check merge FastHashMap and RegularHashMap generate code max size") {
```

Comment:
Why is it ignored? Does it take much time?
Can we check the result, too?

Reply:
The result of this test case is correct and the running time is very short. But I'm not sure if need to open this use case, because I see that other unit test cases of testing CodeGen's maxCodesize are closed. thanks.

Reply:
I see about the code size check.
Would it be possible to add a test to check the result in this case or to let us know which existing test cover this case?